src/Runner.Worker/FileCommandManager.cs

```diff
@@ -259,4 +259,99 @@ public void ProcessCommand(IExecutionContext context, string filePath, Container
             return text.Substring(originalIndex, lfIndex - originalIndex);
         }
     }
+
+    public sealed class SetStepSummaryCommand : RunnerService, IFileCommandExtension
+    {
+        public string ContextName => "step_summary";
+        public string FilePrefix => "step_summary_";
+
+        private const int AttachmentSizeLimit = 128 * 1024;
+
+        public Type ExtensionType => typeof(IFileCommandExtension);
+
+        public void ProcessCommand(IExecutionContext context, string filePath, ContainerInfo container)
+        {
+            if (ShouldUploadAttachment(context, filePath))
```

Comment:
i would suggest not creating extra methods unless you are going to unit test them, if not, just inline all the logic under `ProcessCommand()`.

Reply:
@TingluoHuang I can do that. Just out of curiosity: Is style consistency your main concern in that regard or is there something else I'm missing?

Reply:
style consistency